ScrollingStackViewController/Classes/ScrollingStackViewController.swift

```diff
@@ -8,6 +8,14 @@
 
 import UIKit
 
+public enum InsertionLocation {
```

Comment:
@albertodebortoli-je What are your thoughts on changing "InsertionLocation" to "Position"?